server/views/assess/applications/overview.njk

```diff
@@ -0,0 +1,22 @@
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% from "../../partials/showErrorSummary.njk" import showErrorSummary %}
+
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-l">
+    {{ application.person.name }}
+  </h1>
+
+  <p>This application was submitted on {{ formatDate(application.submittedAt, {format: 'medium'}) }}.</p>
```

Comment:
v minor but I think the date should be in bold

Reply:
oh yes good spot